Userland/Games/GameOfLife/Pattern.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021, Ryan Wilson <ryan@rdwilson.xyz>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Board.h"
+#include <AK/Vector.h>
+#include <LibGUI/Event.h>
+#include <LibGUI/Forward.h>
+
+class Board;
+
+class Pattern {
+public:
+    Pattern(Vector<String>);
+    virtual ~Pattern();
+    Vector<String> get_pattern() { return m_pattern; };
```

Comment:
Please don't use a `get_` prefix for getters, just use the same name as the returned variable, minus the `m_`.